ra2ce/analysis/damages/damage_calculation/damage_network_return_periods.py

```diff
@@ -67,9 +67,9 @@ def __init__(
 
     @classmethod
     def construct_from_csv(
-        cls, path: Path, representative_damage_percentage: float, sep: str = ";"
+        cls, csv_path: Path, representative_damage_percentage: float, sep: str = ";"
```

Comment:
Remove default value?
```suggestion
        cls, csv_path: Path, representative_damage_percentage: float, sep: str
```

Reply:
Ah, missed that one. Adjusted